examples/customresources/models/mynavigation/mynavigation.go

```diff
@@ -23,14 +23,38 @@ func init() {
 	})
 }
 
+// Config is the navigation model's config.
+type Config struct {
+	Lat  *float64 `json:"lat,omitempty"`
+	Long *float64 `json:"long,omitempty"`
```

Comment:
Also need comments for the json tags, and now explicitly what it means with omitempty too.